python-package/lightgbm/sklearn.py

```diff
@@ -1,10 +1,10 @@
 # coding: utf-8
-# pylint: disable = invalid-name, W0105
+# pylint: disable = invalid-name, W0105, C0111
 """Scikit-Learn Wrapper interface for LightGBM."""
 from __future__ import absolute_import
 
 import numpy as np
-from .basic import LightGBMError, Predictor, Dataset, Booster, is_str
+from .basic import LightGBMError, is_str
```

Comment:
self._Booster comes from train(). You can try test_sklearn.py. Pylint helps detect such warnings.